src/main/java/view/TaschenrecherGUI.java

```diff
@@ -1,114 +1,129 @@
 package view;
 
 import javax.swing.*;
+import java.awt.*;
 import java.awt.event.ActionListener;
 
 public class TaschenrecherGUI extends JFrame{
-    private JButton buttonMinus = new JButton("-");
-    private JButton buttonMal  = new JButton("*");
-    private JButton buttonGeteilt = new JButton("/");
-    private JButton buttonWurzel = new JButton("sqert");
-    private JButton buttonZiffer9 = new JButton("9");
-    private JButton buttonZiffer6 = new JButton("6");
-    private JButton buttonZiffer3 = new JButton("3");
-    private JButton buttonKomma = new JButton(",");
-    private JButton buttonQuadrat = new JButton("^2");
-    private JButton buttonZiffer8 = new JButton("8");
-    private JButton buttonZiffer5 = new JButton("5");
-    private JButton buttonZiffer2 = new JButton("2");
-    private JButton buttonZiffer0 = new JButton("0");
-    private JButton buttonZiffer7 = new JButton("7");
-    private JButton buttonWeiterFunktionen  = new JButton("Funktionen");
-    private JButton buttonZiffer4 = new JButton("4");
-    private JButton buttonZiffer1 = new JButton("1");
-    private JButton buttonVorzeichen  = new JButton("(-)");
-    private JButton buttonDEL = new JButton("del");
-    private JButton buttonC = new JButton("C");
-    private JButton buttonPlus  = new JButton("+");
-    private JButton buttonGleich = new JButton("=");
-    private JButton buttonCE = new JButton("CE");
-    private JButton buttonProzent = new JButton("Prozent");
-    private JTextField anzeige = new JTextField();
+    JButton btn1 = new JButton("1");
+    JButton btn2 = new JButton("2");
+    JButton btn3 = new JButton("3");
+    JButton btn4 = new JButton("4");
+    JButton btn5 = new JButton("5");
+    JButton btn6 = new JButton("6");
+    JButton btn7 = new JButton("7");
+    JButton btn8 = new JButton("8");
+    JButton btn9 = new JButton("9");
+    JButton btn0 = new JButton("0");
+    JButton plus = new JButton("+");
```

Comment:
Bitte einheitliche Namen, also auch hier "btnPlus". Und die Namen komplett in Englisch, sonst haben wir da so einen Mischmasch drinne